app/workers/stale_account_worker.rb

```diff
@@ -4,6 +4,7 @@ class StaleAccountWorker
   include Sidekiq::Worker
 
   def perform
-    Account.tenants.suspended_since.find_each(&:schedule_for_deletion!)
+    return if ThreeScale.config.onpremises
```

Comment:
This can be a rolling_updates. We try to remove the on-prem / SaaS differences https://issues.jboss.org/browse/THREESCALE-1022

Reply:
OK can be done as a general task as it is kinda urgent now.